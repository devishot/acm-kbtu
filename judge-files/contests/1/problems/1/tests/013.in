200 100
67 28
110 79
96 163
150 28
122 86
46 16
140 33
39 67
41 122
48 194
199 56
89 95
95 98
11 51
104 127
70 190
144 2
20 43
142 125
67 193
187 165
85 79
45 158
17 27
105 169
26 119
33 138
10 73
146 27
193 74
127 129
82 128
120 84
187 195
85 24
5 150
2 24
119 125
156 13
93 115
134 143
189 113
170 111
162 116
181 123
21 47
107 143
12 156
36 77
83 6
131 50
151 100
16 4
14 190
93 114
164 9
191 45
70 98
125 6
17 195
144 1
61 71
173 154
39 114
178 174
104 60
73 55
176 190
141 177
45 129
85 161
123 98
70 15
133 39
138 37
155 98
39 151
43 173
138 134
3 13
51 55
84 146
41 113
190 12
95 110
18 9
41 14
18 69
117 146
185 88
148 47
165 97
68 163
197 16
158 92
164 133
54 181
137 49
54 148
107 126
