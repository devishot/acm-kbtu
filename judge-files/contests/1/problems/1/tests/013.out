200 199 198 197 196 192 191 189 188 187 186 185 184 183 182 180 179 178 176 175 174 172 171 170 168 167 166 165 164 162 160 159 157 155 153 152 149 147 145 144 142 141 177 140 139 137 136 135 133 132 131 130 124 121 120 118 117 116 112 111 109 108 107 126 106 105 169 104 127 103 102 101 99 97 96 94 93 115 91 90 89 95 110 88 87 85 161 84 146 83 82 128 81 80 79 78 76 75 72 70 68 163 66 65 64 63 62 61 71 60 59 58 57 56 54 181 123 98 148 53 52 50 49 48 194 46 45 158 92 129 44 42 41 122 86 113 40 39 67 193 74 114 151 100 38 36 77 35 34 33 138 37 134 143 32 31 30 29 26 119 125 25 23 22 21 47 20 43 173 154 19 18 69 17 27 195 16 15 14 190 12 156 11 51 10 73 55 9 8 7 6 5 150 28 4 3 13 2 24 1 
